src/locales.json

```diff
@@ -478,5 +478,35 @@
     "delete-post": "Futa posti",
     "hint-post": "Dokeza",
     "private-channel-hint": "🆘 Ukitaka kuongeza watumiaji kwenye chaneli binafsi, wanahitaji kuwa wamesaini ndani ya mojibrag teyari. Kama bado, waambie wajisajili kwanza, na uwajumuishe baada ya hapo!"
+  },
+  "ta": {
+    "flag": "TA",
```

Comment:
Yes, that should work. Update the same in file. Please checkout and let me know wether looks good :)